src/javascript/_common/base/client_base.js

```diff
@@ -451,6 +429,42 @@ const ClientBase = (() => {
         return options_blocked_countries.includes(country);
     };
 
+    const isHighRisk = () => {
+        const landing_companies = State.getResponse('landing_company');
+        const risk_classification = State.getResponse('get_account_status.risk_classification');
+        if (landing_companies) {
+            const financial_company_shortcode = landing_companies.financial_company.shortcode;
+            let gaming_company_shortcode;
+            if (landing_companies.gaming_company) {
+                gaming_company_shortcode = landing_companies.gaming_company.shortcode;
+            }
+            const restricted_countries =
+                financial_company_shortcode === 'svg' ||
+                (financial_company_shortcode === 'svg' && financial_company_shortcode !== 'maltainvest');
```

Comment:
The second condition here is redundant, it's already enough to check for `financial_company_shortcode === 'svg'` or does it mean to be gaming_company instead?

Reply:
oh shoot. you are right. should have been gaming_company